packages/boxel/tests/integration/components/boxel/dropdown-test.ts

```diff
@@ -29,45 +35,77 @@ module('Integration | Component | Dropdown', function (hooks) {
+  test('it errors if it receives a non-button element', async function (assert) {
+    let lastError: Error;
+    Ember.onerror = (e: Error) => {
+      lastError = e;
+    };
 
-    await click(TRIGGER);
+    await render(hbs`
+      <Boxel::Dropdown>
+        <:trigger as |bindings|>
+          <div data-test-dropdown-trigger {{bindings}}>
+            Trigger
+          </div>
+        </:trigger>
+        <:content as |dd|>
+          <button {{on "click" dd.close}}>Close</button>
+        </:content>
+      </Boxel::Dropdown>
+    `);
 
-    assert.dom(DROPDOWN_CONTENT).doesNotExist();
+    assert.strictEqual(
+      lastError!.message,
+      'Only buttons should be used with the dropdown modifier'
+    );
   });
 
-  test('it can open and close on space', async function (assert) {
-    await focus(TRIGGER);
-    // https://w3c.github.io/uievents/#fixed-virtual-key-codes
-    await triggerKeyEvent(TRIGGER, 'keydown', 32);
-
-    assert.dom(DROPDOWN_CONTENT).isVisible();
-
-    await triggerKeyEvent(TRIGGER, 'keydown', 32);
-
+  test('it renders dropdown trigger, but not dropdown content at initialization', async function (assert) {
+    assert.dom(TRIGGER).isVisible().containsText('Trigger');
     assert.dom(DROPDOWN_CONTENT).doesNotExist();
   });
 
-  test('it can open and close on enter', async function (assert) {
-    await focus(TRIGGER);
-    // https://w3c.github.io/uievents/#fixed-virtual-key-codes
-    await triggerKeyEvent(TRIGGER, 'keydown', 13);
+  test('it can open and close on trigger click', async function (assert) {
+    await click(TRIGGER);
 
     assert.dom(DROPDOWN_CONTENT).isVisible();
 
-    await triggerKeyEvent(TRIGGER, 'keydown', 13);
+    await click(TRIGGER);
 
     assert.dom(DROPDOWN_CONTENT).doesNotExist();
   });
 
+  // https://github.com/emberjs/ember-test-helpers/issues/1054
+  // no click event is generated with synthetic keydown
+  // eslint-disable-next-line qunit/no-commented-tests
+  // test('it can open and close on space', async function (assert) {
+  //   await focus(TRIGGER);
+  //   // https://w3c.github.io/uievents/#fixed-virtual-key-codes
+  //   await triggerKeyEvent(TRIGGER, 'keydown', 32);
+  //   assert.dom(DROPDOWN_CONTENT).isVisible();
+  //   await triggerKeyEvent(TRIGGER, 'keydown', 32);
+  //   assert.dom(DROPDOWN_CONTENT).doesNotExist();
+  // });
+
+  // https://github.com/emberjs/ember-test-helpers/issues/1054
+  // no click event is generated with synthetic keydown
+  // eslint-disable-next-line qunit/no-commented-tests
+  // test('it can open and close on enter', async function (assert) {
+  //   await focus(TRIGGER);
+  //   // https://w3c.github.io/uievents/#fixed-virtual-key-codes
+  //   await triggerKeyEvent(TRIGGER, 'keydown', 13);
+  //   assert.dom(DROPDOWN_CONTENT).isVisible();
+  //   await triggerKeyEvent(TRIGGER, 'keydown', 13);
+  //   assert.dom(DROPDOWN_CONTENT).doesNotExist();
+  // });
```

Comment:
Should we keep these? 🤔

Reply:
Seemed to be easier to comment at the rendering error test about the synthetic keydown events not generating clicks, so I've done that instead.